### PR TITLE
GraphQL schema documentation

### PR DESCRIPTION
**Pull request by Schparky**

### CODE REVIEW

File: application/gqlgen/schema.graphql [hunk 1]
```diff
@@ -169,111 +275,151 @@ enum PostVisibility {
...
     country: String!
+    "Latitude in decimal degrees, e.g. -30.95 = 30 degrees 57 minutes south"
     latitude: Float
+    "Longitude in decimal degrees, e.g. -80.05 = 80 degrees 3 minutes west"
     longitude: Float
 }
 
+"Meeting, a/k/a Event, to serve as a focal point for finding, answering, carrying, and exchanging requests"
 type Meeting {
+    "gloablly unique identifier for the `Meeting` object"
```
Comment (Baggerone) [inline application/gqlgen/schema.graphql]:
```suggestion
    "globally unique identifier for the `Meeting` object"
```

File: application/gqlgen/schema.graphql [hunk 2]
```diff
@@ -169,111 +275,151 @@ enum PostVisibility {
...
+    "meeting (event) location -- notifications and filters may use this location"
     location: LocationInput!
-
-    "NOT YET IMPLEMENTED -- Who can see this meeting"
+    "NOT YET IMPLEMENTED -- what subset of users can view and interact with this meeting"
     visibility: MeetingVisibility!
 }
 
 input UpdateMeetingInput {
+    "gloablly unique identifier for the `Meeting` object to be modified"
```
Comment (Baggerone) [inline application/gqlgen/schema.graphql]:
```suggestion
    "globally unique identifier for the `Meeting` object to be modified"
```

File: application/gqlgen/schema.graphql [hunk 3]
```diff
@@ -340,93 +484,155 @@ input RemoveMeetingParticipantInput {
     userID: ID!
 }
 
+"In-app chat message"
 type Message {
+    "gloablly unique identifier for the Message"
```
Comment (Baggerone) [inline application/gqlgen/schema.graphql]:
```suggestion
    "globally unique identifier for the Message"
```

File: application/gqlgen/schema.graphql [hunk 4]
```diff
@@ -169,111 +275,151 @@ enum PostVisibility {
...
     ADMIN
     USER
 }
 
+"""
+File metadata for images and other supported file types. If the URL expiration time passes, a new query will refresh
+the URL and the URL expiration time.
+"""
 type File {
+    "gloablly unique identifier for the `File` object"
```
Comment (Baggerone) [inline application/gqlgen/schema.graphql]:
```suggestion
    "globally unique identifier for the `File` object"
```

**Comment by Schparky** [comment]
> Wow. Lots of work.
> You had the same "gloablly" typo in several places. I made in-line suggested edits, but don't let Codeship create new runs for each of them.

Haha, I actually changed my mind later and searched for "globally" and removed that word. Since these were misspelled I missed them.

**Pull request merged**